### PR TITLE
SUB-1765 | declare AlertChannelAPI struct

### PR DESCRIPTION
**Pull request by refaelm92**
Signed-off-by: refaelm <refaelm@armosec.io>

### CODE REVIEW

File: containerscan/datastructures.go [hunk 1]
```diff
@@ -16,6 +16,13 @@ const (
 	CriticalSeverity   = "Critical"
 
 	ContainerScanRedisPrefix = "_containerscan"
+
+	UnknownScore    = 1
```
Comment (avrahams) [inline containerscan/datastructures.go:20]: No sure this is the right place since severity is not just for containers 
@Bezbran WDYT

Comment (avrahams) [reply containerscan/datastructures.go:20]: I would consider maybe to put these under identifiers or something more general

Comment (refaelm92) [reply containerscan/datastructures.go:20]: do you think we need new file smth like `severities.go` just for those constants?

Comment (avrahams) [reply containerscan/datastructures.go:20]: The package is the main thing not the file.
I was thinking maybe it should be under identifiers next to the other contents - but I see that the map is also here - so let it be

File: notifications/usersnotificationsmethods.go [hunk 2]
```diff
@@ -119,6 +119,30 @@ func (nc *NotificationsConfig) GetLatestPushReport(cluster string, scanType Scan
 	return nil
 }
 
+func (nc *NotificationsConfig) GetAlertChannelByCollaborationID(collaborationId string) (*AlertChannel, error) {
```
Comment (avrahams) [inline notifications/usersnotificationsmethods.go:122]: should we add tests - see the test file we have few for notifications config

**Review by avrahams — APPROVED** [review]

**Pull request merged**